DBMS/Assignment4/zipcode.sql

```diff
@@ -0,0 +1,54 @@
+    -- Creating Database zipcodeInfo
+    CREATE DATABASE zipcodeInfo;
+    USE zipcodeInfo;
+    
+    -- creating table Zipcode
+    CREATE TABLE zipcodeNumber(
+        zipcode int PRIMARY KEY AUTO_INCREMENT
```

Comment:
ACTION: Table structure for zipcodenumber is not correct. According to me there would be 2 columns in this table. first column should be zipcode_id int PRIMARY KEY AUTO_INCREMETED  and second column should be zipcode varchar(10).